### ISSUE TITLE
VK Video support is broken

### ISSUE THREAD

**Issue by veselcraft** [imputnet/cobalt#956]
### bug description

Couple days ago VK registered their new domain "vkvideo.ru" for videos published on VK, and since now it redirects all videos to new domain, and the support of VK is broken.

Example link: 

### reproduction steps

1. paste new link, press dl
2. _your link is invalid or this service is not supported yet. have you pasted the right link?_

1. paste link with vkvideo.ru replaced to vk.com, press dl
2. _couldn't find any media that i could download for you. are you sure you pasted the right link?_

### links

```shell
https://vkvideo.ru/video-4429782_76849466
https://vkvideo.ru/video-1719791_456263870
https://vkvideo.ru/video-25016515_456240870
```

### platform information

Zen Browser 1.0.1-a.19 (Based on Firefox 132.0.1) on Windows 11

**Comment by wukko** [comment]
already implemented on the dev branch: https://github.com/imputnet/cobalt/commit/d4bcb1ba617ad0f099b23e03ce05e4d41895ba9f

will close this issue once it’s merged

**Comment by wukko** [comment]
pretty sure something's up on vk's end though

https://github.com/user-attachments/assets/80b4d727-3681-4931-a927-a850086906c4

**Comment by veselcraft** [comment]
womm

**Comment by wukko** [comment]
closed via #964
